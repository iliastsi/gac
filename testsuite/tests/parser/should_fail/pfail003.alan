hello() : proc
 c : int;
{
 (* unknown token *)
 c = $ 4;
}
