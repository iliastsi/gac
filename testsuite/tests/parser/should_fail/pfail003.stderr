pfail003.alan:5:6: Error: Parse error on input `$'
	Cannot parse char `$'

